server/chat-commands/admin.ts

```diff
@@ -943,11 +949,12 @@ export const commands: ChatCommands = {
 			} else {
 				result = Utils.visualize(result);
 			}
-			result = result.replace(/\n/g, '\n||');
-			this.sendReply('||<< ' + result);
+			logRoom?.roomlog(`<< ${result}`);
+			this.sendReply(`|html|<table><tr><td>&laquo;</td><td>${Chat.getReadmoreCodeBlock(result)}</td></tr><table>`);
```

Comment:
```suggestion
			this.sendReply(`|html|<table><tr><td>&laquo;</td><td>${Chat.getReadmoreCodeBlock(result)}</td></tr><table>`);
			logRoom?.roomlog(`<< ${result}`);
```

(All the other roomlogs are now after the `sendReply`, so we might as well make this consistent.)